lms/static/sass/shared-v2/_base.scss

```diff
@@ -22,3 +22,30 @@
 .sr {
     @extend .sr-only;
 }
+
+.site-status {
+  padding: ($baseline/2);
+  @include linear-gradient(top, $shadow-l1, rgba(0, 0, 0, .0));
+  background-color: $site-status-color;
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, .3) inset;
+  font-size: 14px;
+
+  .icon {
+    @include margin-right($baseline / 2);
+    @include float(left);
+    color: $white;
+    font-size: 18px;
+  }
+
+  .inner-wrapper {
+    margin: auto;
+    max-width: 1180px;
+    min-width: 760px;
+  }
+
+  p {
+    margin-bottom: 0;
+    line-height: 1.3;
+    color: $white;
+  }
+}
```

Comment:
Lines 25-51 here are almost the exact same as lines 198-223 in `base/_base.scss` above.

a) is there any to get rid of this duplication? I know our v1/v2 sass situation is less than ideal, but maybe a shared import?
b) as far as _differences_ between those two CSS blocks, `.site-status` is `display: none` in `base/_base.scss` but not in `shared-v2/_base.scss`?